.github/workflows/release.yml

```diff
@@ -34,4 +34,5 @@ jobs:
         TWINE_PASSWORD: ${{ secrets.pypi_password }}
       run: |
         python setup.py sdist
+        python setup.py bdist_wheel
```

Comment:
I think this is good for now.  I do think it's worth switching this repo over to using `pyproject.toml` and using `build` to create the source distribution and wheel, again [like how the CircuitPython libraries do](https://github.com/adafruit/workflows-circuitpython-libs/blob/a2776de920417999245a2f2688d15f125eaa44a4/release-pypi/action.yml#L42).

It's probably worth having the `bdist_wheel` here use the `--universal` flag, or remove it in `build.yml`.

Reply:
Thinking more about it, let's just add the `--universal` flag here for consistency.

Reply:
Yeah, that'd be nice, but I didn't want to change too much.

About `--universal` I'll remove it in the build script, as this is a pure python package and it's a bit confusing that this flag is used (I did check when I first saw it, in case there was some non-python code I didn't notice before).

Reply:
> Thinking more about it, let's just add the --universal flag here for consistency.

Sorry, the GH page didn't refreshed before I made the change. Would you prefer that I add it in both? As mentioned, this is a pure python package, and apart from being unnecesary, it's a bit confusing that it's there.

Reply:
Yeah, it's pure Python, and while I originally thought it would be nice to catch things by using `--universal`, it will be caught in a PR review (if that were to ever even happen, which it realistically will not).  Let's just remove it, as you suggested.